code/game/objects/structures/arclumin.dm

```diff
@@ -0,0 +1,21 @@
+/*
+
+At 7:30 AM on March 8th, 2017, ArcLumin died in a fatal car crash on impact.
```

Comment:
Anonymized as it is, this would be a proper addition even if this event *hadn't* happened.  Plus that is a real cool sprite.
There are many people's usernames in many comments throughout SS13 code.  None of that will appear in game to bother strangers.